dev-python/flask-peewee/flask-peewee-0.6.5-r1.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
Do the usual `git mv` here, and retain stable keywords. This is only a dep update, right?